### PR TITLE
Make project list in Solution Package Manager resizable.

### PR DESCRIPTION
**Pull request by feiling**
@yishaigalatzer @emgarten @deepakaravindr

### CODE REVIEW

File: src/NuGet.Clients/PackageManagement.UI/Converters/ProjectAndSolutionViewMinHeightConverter.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+namespace NuGet.PackageManagement.UI 
+{
+    class ProjectAndSolutionViewMinHeightConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var isSolution = (bool)value;
+            if (isSolution)
+            {
+                return 200.0;
```
Comment (yishaigalatzer) [inline src/NuGet.Clients/PackageManagement.UI/Converters/ProjectAndSolutionViewMinHeightConverter.cs:19]: how does the 200 (and the 300) above work on different resolutions?

**Comment by yishaigalatzer** [comment]
:shipit:

File: src/NuGet.Clients/PackageManagement.UI/Converters/ProjectAndSolutionViewMinHeightConverter.cs (hunk 1 above)
Comment (feiling) [inline src/NuGet.Clients/PackageManagement.UI/Converters/ProjectAndSolutionViewMinHeightConverter.cs:19]: This number is device independent pixel so the size will be adjusted by WPF based on the current resolution.

**Comment by feiling** [comment]
Fix merged.

**Pull request closed without merging**